### PR TITLE
Docstring edits

### LINKED ISSUE #203: Docstring edits
I'm offering to edit docstrings in a new PR. I think I saw that this repo encourages opening issues before PRs.

**Describe the bug**

Docstrings have a variety of minor issues. Some examples:
- docstring has wrong parameter name ([example](https://github.com/dreamquark-ai/tabnet/blob/develop/pytorch_tabnet/metrics.py#L281))
- indentation isn't standard NumPy format ([example](https://github.com/dreamquark-ai/tabnet/blob/develop/pytorch_tabnet/abstract_model.py#L96))
- return value isn't listed ([example](https://github.com/dreamquark-ai/tabnet/blob/develop/pytorch_tabnet/abstract_model.py#L278))

**What is the current behavior?**

N/A

**If the current behavior is a bug, please provide the steps to reproduce.**

N/A

**Expected behavior**

N/A

**Screenshots**

N/A

**Other relevant information:**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by jrfiedler**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

Issue #203

**What kind of change does this PR introduce?**

Mostly docstring edts.

In a few places I've added info to docstrings, and those should probably get some extra checking to make sure they're correct.

In one place I've changed a parameter name (see next question).

**Does this PR introduce a breaking change?**

This currently changes one function parameter name. This is probably the most controversial change here. I'm happy to change it back if the old name is preferred.

In `abstract_model.py`, in the method `TabModel.save_model()`, I've changed `path` to `filepath`. I made this change because the following method, `TabModel.load_model()` uses `filepath`, and there are common submodules in the standard library called `path` (`os.path`, `sys.path`). Also, I didn't see any code that references the parameter name in .py or .ipynb files. But, again, I'm happy to change this back if the old name is preferred.

**What needs to be documented once your changes are merged?**

Nothing. The changes here are mostly in the documentation.

**Closing issues**

closes #203

### CODE REVIEW

**Comment by eduardocarvp** [comment]
Thanks, @jrfiedler ! 
I had a quick look on the changes and it should be good. I just need to test it along with the Sphinx document generation and I think we can merge.

The variable name change seems ok, I personally prefer `filepath` as well.

File: pytorch_tabnet/utils.py [hunk 1]
```diff
@@ -58,24 +58,32 @@ def create_dataloaders(
...
+    batch_size : int
+        how many samples per batch to load
+    num_workers : int
+        how many subprocesses to use for data loading. 0 means that the data
+        will be loaded in the main process
+    drop_last : bool
+        set to True to drop the last incomplete batch, if the dataset size is not
+        divisible by the batch size. If False and the size of dataset is not
+        divisible by the batch size, then the last batch will be smaller
+
```
Comment (Optimox) [inline pytorch_tabnet/utils.py:82]: Could you add `pin_memory` please?
It's a boolean value that decides whether to pin gpu memory during training.

**Review by Optimox — CHANGES_REQUESTED** [review]
Thanks!
Did you pass through all the code?

**Comment by jrfiedler** [comment]
@Optimox, I've reverted the parameter name and I added `pin_memory` to that docstring.

And yes, I have looked through all of the Python files for issues like these, and I fixed all that I found.

**Comment by Optimox** [comment]
@jrfiedler could you squash all your commits into one please? Then it's good to go!

**Review by Optimox — APPROVED** [review]

**Comment by Optimox** [comment]
Thank you very much @jrfiedler !

**Pull request merged**